excellent/evaluator.go

```diff
@@ -317,6 +318,10 @@ func EvaluateTemplateAsString(env utils.Environment, resolver utils.VariableReso
 				buf.WriteString(token)
 			} else {
 				strValue, _ := utils.ToString(env, value)
+				if urlEncode {
```

Comment:
same as in old engine - need this for when the expressions should be URL encoded, but the template itself shouldn't